proteomics/mPXD018682/README.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Metaproteomics of PXD018682
+title: Metaproteomics of PXD019423
```

Comment:
@subinamehta is that a new dataset? If so I need to change the folder name and a few other references to this page.